tests/bluetooth/tester/src/btp/btp_ascs.h

```diff
@@ -19,8 +19,8 @@ struct btp_ascs_configure_codec_cmd {
 	uint8_t coding_format;
 	uint16_t vid;
 	uint16_t cid;
-	uint8_t ltvs_len;
-	uint8_t ltvs[0];
+	uint8_t codec_ltvs_len;
+	uint8_t codec_ltvs[0];
```

Comment:
```suggestion
	uint8_t codec_cfg_ltvs_len;
	uint8_t codec_cfg_ltvs[0];
```
or
```suggestion
	uint8_t cc_ltvs_len;
	uint8_t cc_ltvs[0];
```

Codec in this context is a bit too generic IMO